src/pages/careers/besquare/components/sections/_hero.tsx

```diff
@@ -1,11 +1,41 @@
 import React from 'react'
+import styled from 'styled-components'
 import { HeroContainer, HeaderContainer, ImageWrapper } from '../../static/style/_hero'
 import { TextWrapper, Title } from '../../static/style/_common'
 import HeroImage from 'images/common/be-square/hero.png'
+import device from 'themes/device'
+
+export const StyledHeaderContainer = styled(HeaderContainer)`
+    padding: 120px 0 80px 120px;
+
+    @media ${device.desktop} {
+        padding: 40px 0 0 0;
+    }
+
+    @media ${device.laptop} {
+        padding: 40px 0 0;
+    }
+`
```

Comment:
Why are we styling this here instead of in `HeaderContainer`?

Reply:
I thought it will affect other  components previously so I decided to add new custom component